e2e/healthcheck_test.go

```diff
@@ -0,0 +1,30 @@
+package e2e_test
+
+import (
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("healthz check should pass", func() {
+	Context("start server with health check", func() {
+		It("should work properly", func() {
+			address, ok := os.LookupEnv("kepler_address")
+			if !ok {
+				address = "localhost:8888"
+				cmd := exec.Command(keplerBin)
+				keplerSession, err = gexec.Start(cmd, nil, nil)
```

Comment:
Will the test start Kepler alone or with the other components, like the Model Server?

If it is only Kepler, what is the integration test part?

I am not saying that this test here is not important, but it might be not integration test as the title suggest...

Reply:
In the case it is not integration test, this is definitely a functional test that we need to have